### DISCUSSION TITLE
Sensorerkennung HmIP-ESI

### DISCUSSION THREAD

**Discussion by He1no** [OpenCCU/OpenCCU#2850] [🇩🇪 Fragen&Antworten]
Hallo, ich habe meinen Gasverbrauchszähler HmIP-ESI über RaspberryMatic in mein Home Assistant eingefügt. 

Er zeigt aber die ganze Zeit an das die Leistung unbekannt ist und der Kanal keine Funktion hat.

![CleanShot 2024-09-15 at 09 22 32@2x](https://github.com/user-attachments/assets/b2d27845-caf6-432a-9b6e-387c2d406dd3)

Klicke ich oben auf den Sensor, kann ich sehen, das ich noch eine Sensorerkennung machen soll. 

![CleanShot 2024-09-15 at 09 24 50@2x](https://github.com/user-attachments/assets/9f814b61-0e7c-47b3-9d31-60e8b186bf8b)

Wenn ich darauf klicke und dann auf dem HmIP-ESI auf die einzige Taste klicke die er hat, passiert genau gar nichts. 
Kann mir jemand sagen was ich da falsch mache ?

**Comment by Baxxy13** [comment]
Wende dich mal an das [Homematic-Forum](https://homematic-forum.de/).
Da sind mehr Leute unterwegs die Dir ggf. helfen können.

**Reply by He1no** [reply]
Ok alles klar ich versuch da mal mein Glück
